src/hardware/opl.h

```diff
@@ -59,87 +58,86 @@ class Timer {
 		SetCounter(0);
 	}
 
-	//Update returns with true if overflow
-	//Properly syncs up the start/end to current time and changing intervals
+	// Update returns with true if overflow
+	// Properly syncs up the start/end to current time and changing intervals
 	bool Update(const double time)
 	{
 		if (enabled && (time >= trigger)) {
 			// How far into the next cycle
 			const double deltaTime = time - trigger;
 			// Sync start to last cycle
 			const auto counterMod = fmod(deltaTime, counterInterval);
-			start = time - counterMod;
-			trigger = start + counterInterval;
-			//Only set the overflow flag when not masked
+			start                 = time - counterMod;
+			trigger               = start + counterInterval;
+			// Only set the overflow flag when not masked
```

Comment:
Maybe an empty line separating the two constants versus the two members (logically separate their different roles, and the right-alignment would tighten up for `start` and `trigger`).